src/internal.c

```diff
@@ -29285,13 +29416,21 @@ int SendClientKeyExchange(WOLFSSL* ssl)
                     peerKey = (ssl->specs.static_ecdh) ?
                               ssl->peerEccDsaKey : ssl->peerEccKey;
 
-                    ret = EccSharedSecret(ssl,
-                        (ecc_key*)ssl->hsKey, peerKey,
-                        args->encSecret + OPAQUE8_LEN, &args->encSz,
-                        ssl->arrays->preMasterSecret,
-                        &ssl->arrays->preMasterSz,
-                        WOLFSSL_CLIENT_END
-                    );
+                    #ifdef WOLFSSL_MAXQ10XX_TLS
+                    if ((ssl->hsKey) &&
+                        (((ecc_key*)ssl->hsKey)->maxq_ctx.hw_storage == 1)) {
+                        WOLFSSL_MSG("Skipping EccSharedSecret()");
```

Comment:
I would modify this log to indicate that it's being skipped because WOLFSSL_MAXQ10XX_TLS is defined. Just more descriptive.

Reply:
This code has been removed.